test/webdriver/test-link-clicking.js

```diff
@@ -0,0 +1,127 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+/* eslint prefer-arrow-callback: 0 */
+
+import expect from 'expect';
+import { quickTest } from '../../build-config';
+import Driver from '../utils/driver';
+import * as selectors from '../../app/ui/browser/selectors/index';
+
+const STARTING_TAB_COUNT = 1;
```

Comment:
In other tests, I regret having this as it makes the assertions harder to follow -- I think we can assume one tab on opening for these tests